OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -291,11 +292,79 @@ public override bool HandleKeyPress(KeyInput e)
 					return ToggleMute();
 				else if (key == Game.Settings.Keys.TogglePlayerStanceColorsKey)
 					return TogglePlayerStanceColors();
+				else if (key == Game.Settings.Keys.NextTrack)
```

Comment:
You could add a LogicKeyListener widget to the yaml and then attach logic to its OnKeyPress handler.